styles/style.css

```diff
@@ -60,29 +63,32 @@
 }
 
 .header__subtitle{
+  margin: 0 auto;
+  padding: 0;
   position: absolute;
   font-size: 18px;
   width:388px;
-  margin: 0 auto;
-  padding: 0;
   bottom: 30px;
   left: 64px;
   z-index: 1;
 }
 
 .header__link{
+  margin: 0 auto;
+  padding: 0;
   text-decoration: none;
   font-size: 18px;
   color: #2F80ED ;
 }
 
-
 .header__link:hover{
-opacity: .7;
-transition: opacity .3s ease;
+  opacity: .7;
+  transition: opacity .3s ease;
```

Comment:
The `transition` property has an interesting behavior - if it's set for an element from the start, it will create smooth animations _for any state change_, not just on hover.

Right now, the smooth transition is only applied when you hover over the link. When you move the cursor away, the transition becomes abrupt. 

To make it work smoothly in both directions, move the `transition` property from the `:hover` state to **the element itself**. This way, the animations will be consistent and pleasant for users.